application-collabora-api/src/main/resources/xwiki.collabora.rest.model.xsd

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ *
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ *
+-->
+<schema xmlns="http://www.w3.org/2001/XMLSchema" targetNamespace="http://www.xwiki.com/collabora"
+  xmlns:collabora="http://www.xwiki.com/collabora" elementFormDefault="qualified">
+
+  <complexType name="Token">
+    <annotation>
+      <documentation>Stores a token metatada.</documentation>
+    </annotation>
+    <sequence>
+      <element name="value" type="string" minOccurs="0"/>
+      <element name="urlSrc" type="string" minOccurs="0"/>
+      <element name="usage" type="int" minOccurs="0"/>
```

Comment:
Maybe add also ``maxOccurs="1"``?

Reply:
`maxOccurs="1"` is the default value, so I think it's redundant